### ISSUE TITLE
updateStatus not being set properly in Helm-based install?

### ISSUE THREAD

**Issue by winterhalder** [bitnami-labs/sealed-secrets#975]
**Which component**:
Controller v0.18.4, from Helm Chart 2.6.8, 

**Describe the bug**
status is not being set on sealed-secret.
Seems like updateStatus=true is not being properly applied from the Helm Chart?

**To Reproduce**

1. Install sealed-secrets from helm chart
2. Create a sealed secret and apply it to the cluster
3. Describe the sealed secret - see that status is not set.

**Expected behavior**
status field should be set.

**Version of Kubernetes**:
Tested in 1.24 and 1.21

- Output of `kubectl version`:
```
Client Version: version.Info{Major:"1", Minor:"22", GitVersion:"v1.22.13", GitCommit:"a43c0904d0de10f92aa3956c74489c45e6453d6e", GitTreeState:"clean", BuildDate:"2022-08-17T18:28:56Z", GoVersion:"go1.16.15", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"21+", GitVersion:"v1.21.14-eks-18ef993", GitCommit:"ac73613dfd25370c18cbbbc6bfc65449397b35c7", GitTreeState:"clean", BuildDate:"2022-07-06T18:06:50Z", GoVersion:"go1.16.15", Compiler:"gc", Platform:"linux/amd64"}
```

**Additional context**
Seems to work properly when I no a manual (non-Helm) install, but I'm trying to make things more repeatable / upgradeable via Helm.

I've tried setting updateStatus from the Helm command a few different ways -- but really, this is supposed to be the new default, so that shouldn't be necessary at all - right?

```
# Install kubeseal
helm upgrade --install \
	--reset-values \
	--namespace=kube-system \
	--set-string fullnameOverride=sealed-secrets-controller \
	--set updateStatus=true \
	--set 'commandArgs[0]=--update-status' \
	--version $KUBESEAL_HELM_VERSION \
sealed-secrets sealed-secrets/sealed-secrets
```

Note - KUBESEAL_HELM_VERSION is set to "v2.6.8" in my most recent attempt.
![2022-09-22_14-11-52](https://user-images.githubusercontent.com/3590310/191822329-47e2c3e7-306d-472b-abf7-c31d0f009f0e.png)

**Comment by alemorcuq** [comment]
Hi, @winterhalder.

I'm not able to reproduce the issue. Here's what I did:

1. Install chart version 2.6.8 with your provided command:

```
$ helm upgrade --install \
        --reset-values \
        --namespace=kube-system \
        --set-string fullnameOverride=sealed-secrets-controller \
        --set updateStatus=true \
        --set 'commandArgs[0]=--update-status' \
        --version 2.6.8 \
sealed-secrets sealed-secrets/sealed-secrets
```

2. Create and apply a Sealed Secret:

```
$ kubectl create secret generic test --dry-run=client --from-literal foo=bar -o yaml | kubeseal --controller-name sealed-secrets-controller -o yaml | kubectl apply -f -
sealedsecret.bitnami.com/test created
```

3. Describe the Sealed Secret:

```
$ kubectl describe sealedsecrets.bitnami.com test
...
Spec:
  Encrypted Data:
    Foo:  AgAyWW+dt2AzVqj3XDZb0EqVruyFC4zAX7KFXCa0Y4hicduXNV4L6GDLN+5OYE/7r3wLwoY/CXhMTxjT4ohMgZpX6s9d+VmrPiEf961F8rxHWPL6h5pQ5LZp/Y5oTY+i1s1VlHOTLKabpOxFMqMax5RoDnMbIiBr2dMK8V95vBBP7fykNxgJcTLc1OjIxNOtOYqz+8OpWItmYToGNtNlbp9FcOG6yPm7rBYMEfXPkVBGo3LBhQYCjJEe31kNsfDqet7+e5iW0Yzd0PmsV5Pgz7GDECT8pdsClGxYHSQO+HKuYn5peV2qcouaIQeJnXOval08+Wzv3fog6xAn/wRkv3XfgXGr0zVtwmpwI0IxL+FAy9BPSxBqPQ6Bez5q/raDEWzBj1t0N/sBnBQc8MJABmht7o4AHPuZE96dxjVZDORuVZ6dOg66BhsjezlZ+2um8AoOqaAFXQS2Rz4OykeNwD40ukIkHbh4xoZiarntzlj+dfoz4plMjjeU8b9JCzxpDTVUvfoSt9QhTzL+5gtUFIxDvOeuGtiVfdD1RKlpRYuDdqxPncoTlrkSoUm2pI8o3lkkMuqoXrIPoVqiMNqya61X4ewtcOTFfVQ9jMHn5MaLeb2/W939LibTf9bN0P2QwKIv16qt8aXr48h22rC9092iaqnWI/VZTqMpmyi/686KaDwVS6F9TWDb7HxSLVxdFh+dMyE=
  Template:
    Data:  <nil>
    Metadata:
      Creation Timestamp:  <nil>
      Name:                test
      Namespace:           default
Status:
  Conditions:
    Last Transition Time:  2022-09-23T08:59:53Z
    Last Update Time:      2022-09-23T08:59:53Z
    Status:                True
    Type:                  Synced
  Observed Generation:     1
Events:
  Type    Reason    Age              From            Message
  ----    ------    ----             ----            -------
  Normal  Unsealed  4s (x2 over 4s)  sealed-secrets  SealedSecret unsealed successfully
```

As you can see, the status is there. 

We recently had an issue with the status subresource when using the chart (#961) but it was fixed in (#964). The issue lied in the CRD. Could you please double check you are using the updated CRD? It may be possible you have an older version installed in your cluster.

**Comment by winterhalder** [comment]
Ah - uninstalling the helm chart, then uninstalling the CRDs, then re-installing the helm chart - and finally recreating the secrets did it... 

Thanks a bunch! Apologies for the erroneous report!
